react-big-calendar/index.d.ts

```diff
@@ -48,5 +50,9 @@ declare module 'react-big-calendar' {
         static globalizeLocalizer(globalizeInstance: Object): void;
     }
 
-     export default BigCalendar;
+    /* This enables 'import * as BigCalendar' syntax when compiling to es2015 */
+    namespace BigCalendar {}
+
+    /* react-big-calendar is exported as a commonjs module (it uses babel-preset-jason) */
+    export = BigCalendar;
```

Comment:
To be more precise of course you still will be able to use `import * as BigCalendar`, but to use it correctly you have to access default property from namespace like this:
```
import * as BigCalendar
const BigCalendarDefault = BigCalendar.default;
```